src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -803,4 +836,12 @@ class LicenceService(
 
     return newLicence
   }
+
+  private fun splitName(fullName: String?): Pair<String?, String?> {
+    val names = fullName?.split(" ")?.toMutableList()
+    val firstName = names?.firstOrNull()
+    names?.removeAt(0)
```

Comment:
super unnecessary comment (I know you're just moving this method!)

`drop(1).join...` would have got rid of the mutable list.